### ISSUE TITLE
Configure source in JSON

### ISSUE THREAD

**Issue by beruic** [madskristensen/Packman#23]
It would be nice if you could configure the source for packages in `packman.json`, and perhaps have a section per source and one for custom sources
It would:
- naturally order your packages after source
- save the trouble of explaining to new developers that they should change the settings of the extension
- render the configuration unnescesary

**Comment by madskristensen** [comment]
Yeah, i've been thinking about this, but I don't know if the source should be visible to the user at all. I'm not sure it matters if jQuery.js comes from JSDelivr or CDNJS, so why even show it - let alone make it configurable like it is now?

**Comment by beruic** [comment]
Yes. The thing is that the content of the downloaded package differs between the two sources, and so does the name of some packages (e.g. bootstrap@JsDelivr vs twitter-bootstrap@CDNJS. This is very common in fact). Some packages are entirely unavailable from one of the sources, or not available in the same versions.

This also means that if your project uses packages from CDNJS, all developers must configure the extension to use CDNJS instead of JsDelivr. Otherwise you will get errors in your packman.json or some files may be unavailable from some packages.
